### ISSUE TITLE
nn modules to prioritize

### ISSUE THREAD

**Issue by skeydan** [mlverse/torch#145]
These would be great to have next :-))

```
nn_batch_norm1d
nn_transpose_conv2d
nn_leaky_relu
```

merci :-)

**Comment by dfalbel** [comment]
With #153 they will be all available.

**Comment by skeydan** [comment]
awesome, thanks!

**Comment by jwijffels** [comment]
don't know if I can hijack this issue but what about adding 
nn.Embedding and nn.Parameter

**Comment by skeydan** [comment]
sorry, typo from my side :-(

should have been `batchnorm2d` ;-))

**Comment by skeydan** [comment]
and thanks :-)
